application/event_processor/dos.py

```diff
@@ -197,13 +198,13 @@ def _connect_dos_db() -> connection:
 
     warning: Do not use. Should only be used by query_dos_db() func
     """
-
+    db_secret = get_secret(environ["DB_SECRET_NAME"])
     server = environ["DB_SERVER"]
     port = environ["DB_PORT"]
     db_name = environ["DB_NAME"]
     db_schema = environ["DB_SCHEMA"]
     db_user = environ["DB_USER_NAME"]
-    db_password = environ["DB_PASSWORD"]
+    db_password = db_secret[getenv("DB_SECRET_KEY")]
```

Comment:
Think we might want to use **environ** here instead of **getenv** to get the environment variable for consistents with how we're loading it elsewhere. Line 201 above uses **environ**

Reply:
good spot. make sense